bin/hdfcoinc/pycbc_multiifo_dtphase

```diff
@@ -1,70 +1,69 @@
 #!/bin/env python
-""" Create a file containing the time, phase, and amplitude 
-correlations between two or more detectors by
+""" Create a file containing the phase and amplitude, 
+correlations between two detectors by
```

Comment:
what happened to this description, seems to have become less accurate (failure to rebase previous changes?)